src/TestUtils/src/UITest.Appium/HelperExtensions.cs

```diff
@@ -494,7 +494,16 @@ public static void DragCoordinates(this IApp app, float fromX, float fromY, floa
 			});
 		}
 
-		static IUIElement Wait(Func<IUIElement> query,
+		/// <summary>
+		/// Navigate back on the device.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		public static void Back(this IApp app)
```

Comment:
Back
https://learn.microsoft.com/en-us/dotnet/api/xamarin.uitest.ios.iosapp.back?view=xamarin-uitest-sdk

Reply:
Notice that we already have a Back method in the `UtilExtensions` class. However, this method try to find the visual back button and click it. This use the physical back button on Android etc.